app/views/customers/index.html.erb

```diff
@@ -1,18 +1,18 @@
-<h1>Listagem de clientes</h1>
+<h1><%= t('.title') %></h1>
 
 <table>
   <thread>
     <tr>
-      <th>Nome</th>
-      <th>CPF</th>
+      <th><%= t('.list.name') %></th>
+      <th><%= t('.list.cpf') %></th>
```

Comment:
This is a solution, but I think that field doesn't belongs to the attribute name, but something like the table field. Like the example https://github.com/rafaelrpbelo/car_dealership_sm/blob/internationalize-content/app/views/vehicles/index.html.erb#L8

I think this way I can keep a standard easily. But your suggestion is too welcome.